sdk/maps/ci.yml

```diff
@@ -44,3 +44,5 @@ extends:
       safeName: AzureMapsSearch
     - name: Azure.Maps.Geolocation
       safeName: AzureMapsGeolocation
+    - name: Azure.Maps.Weather
+      safeName: AzureMapsWeather
```

Comment:
- We should remove these 2 lines before merging. Otherwise, this can block other packages to ship.